.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,38 +1,10 @@
-<!--
-Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+### What / Why
```

Comment:
What it solves? Why?

Reply:
same here.

Reply:
`Why` might sound a bit passive-aggressive. Let's just stick with `What it solves`.

Reply:
done